.travis.yml

```diff
@@ -45,8 +45,8 @@ before_install:
   - python -c "import sys; print(sys.executable)"
   - python -m pip --version
   - pip --version
-  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then virtualenv venv --python=python3; fi
-  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then source venv/bin/activate; fi
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then virtualenv /tmp/venv --python=python3; fi
+  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then source /tmp/venv/bin/activate; fi
```

Comment:
Good catch here! I guess we never realised PyFlakes was going crazy analysing this directory.

Reply:
If I understand the current implementation, `./venv` is being excluded in the `find ` command options.